roles/pulp_database_config/tasks/main.yml

```diff
@@ -264,7 +264,8 @@
   changed_when: result.rc == 0
   failed_when: >
     (result.rc not in [0,1]) or
-    ((result.rc == 1) and ('Key (name)=(ansible-default) already exists.' not in result.stderr))
+    ((result.rc == 1) and
+    ('Key (name)=({{ __pulp_common_merged_pulp_settings.galaxy_collection_signing_service }) already exists.' not in result.stderr))
```

Comment:
I believe that we are missing a '}'  here:
`{{ __pulp_common_merged_pulp_settings.galaxy_collection_signing_service }`

Reply:
Good catch! Fixed.